src/plugins/mer/meremulatordevice.cpp

```diff
@@ -197,6 +197,7 @@ MerEmulatorDevice::MerEmulatorDevice(Core::Id id)
     , m_cpuCount(0)
     , m_vdiCapacityMb(0)
 {
+    init();
```

Comment:
The upstream moved away from having separate init() in commit 54385ad67be075ac14d0edae5c451ed3d925edc8, maybe we should follow their example?

Reply:
Added commit 98780c6cab3880c179c8f45d14b1c6733d1b9b99 but decided to left init() there to not break history (it is not immediately adjacent to the constructor)